client/my-sites/customer-home/controller.jsx

```diff
@@ -67,7 +68,17 @@ export async function maybeRedirect( context, next ) {
 			checklist: launchpadChecklist,
 		} = await fetchLaunchpad( slug );
 
+		const experimentAssignment = await loadExperimentAssignment(
+			'calypso_onboarding_launchpad_removal_test_2024_08'
+		);
+
+		const shouldShowLaunchpad =
+			'treatment' !== experimentAssignment?.variationName &&
+			// for testing/development purposes we can use sessionStorage to force the treatment
+			'treatment' !== window.sessionStorage.getItem( 'launchpad_experiment_variation' );
```

Comment:
It may be worth using a more specific name to avoid accidental re-use.
```suggestion
			'treatment' !== window.sessionStorage.getItem( 'launchpad_removal_2024_experiment_variation' );
```

Reply:
I agree with you. I'll update it here and in [the calypso PR](https://github.com/Automattic/jetpack/pull/39237) as well.